### PR TITLE
Make tracker more generic

### PR DESCRIPTION
**Pull request by hmgibson23**
Also add a cljs build for the frontend

### CODE REVIEW

File: app/data/Tracker.scala [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import com.amazonaws.services.dynamodbv2.model.GetItemResult
+import com.amazonaws.services.dynamodbv2.model.ScanRequest
+import com.amazonaws.services.dynamodbv2.model.{AttributeValue,PutItemRequest, PutItemResult}
+
+import java.time.{Instant, ZoneOffset}
+import java.time.format.DateTimeFormatter
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
```
Comment (jamespamplin) [inline app/data/Tracker.scala]: I think play has its own global `ExecutionContext`, which may be better to use. The normal global should work fine though.

File: cloudformation/tracker.json [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+            "Description": "Ip range for the office",
+            "Type": "String",
+            "Default": "77.91.248.0/21"
+        }
+    },
+    "Mappings": {
+        "SSLCertificateId": {
+            "Environment": {
+                "CODE": "arn:aws:iam::743583969668:server-certificate/sites.code.dev-gutools.co.uk-exp2023-08-15",
+                "PROD": "arn:aws:iam::743583969668:server-certificate/sites.gutools.co.uk-exp2015-10-20"
```
Comment (jamespamplin) [inline cloudformation/tracker.json]: @sihil has a new cert, might be worth doing this sooner rather than later.

**Comment by jamespamplin** [comment]
:+1: nice work! Excited to see some clojurejs in here :)

File: app/data/Tracker.scala (hunk 1 above)
Comment (jamespamplin) [inline app/data/Tracker.scala]: `import play.api.libs.concurrent.Execution.Implicits._`

**Comment by hmgibson23** [comment]
Am going to merge this and fix the other things later

**Pull request merged**